inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -16,6 +17,53 @@ LayerTestsCommon::LayerTestsCommon() : threshold(1e-2f) {
 void LayerTestsCommon::Run() {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
 
+    if (targetDevice == CommonTestUtils::DEVICE_GNA) {
```

Comment:
@mikhail-treskin i think this is no appropriate changes in common tests, GNA scale factors are pure specifics of backend
Please move this into something GNA common layer tests if you need common, but again i woug disagree in scale factors hardcoding - it is specific for every test - for example i dont need them if to use FQ layers in subgraphs

Reply:
We had an offline discussion related to the same question. We are waiting changes in source code

Reply:
we already agreed on direction. I'm prototyping a solution

Reply:
[related PR](https://github.com/openvinotoolkit/openvino/pull/2731) was merged to the master